database/buildSchema/45_activity_log.sql

```diff
@@ -364,7 +366,8 @@ BEGIN
                                             assigned_sections
                                         FROM review_assignment
                                     WHERE
-                                        id = rev_assignment_id))
+                                        id = rev_assignment_id
+                                        AND assigned_sections <> '{}'))
```

Comment:
So much clearer and easier to see the diff! (And now I see it's actually a very small change). 

Thanks.

One thing -- you'll need to re-copy and paste this into the migration script.

Reply:
NP. I'll do this now